tests/schema.py

```diff
@@ -211,7 +211,7 @@ def add(parser):
         package="samples/apps/logging/liblogging",
         nodes=infra.e2e_args.min_nodes(cr.args, f=0),
         initial_user_count=1,
-        ledger_chunk_bytes=1,  # Chunk ledger at every signature transaction
+        ledger_chunk_bytes="1",  # Chunk ledger at every signature transaction
```

Comment:
Should that become ledger_chunk="1B"?

Reply:
This also works when omitting the unit suffix but I'll add it here for clarity.

Reply:
Is there any way to make not work? I think we'd decided it was better to be explicit as much as possible. Removing the unit from the names also seems good to do now if we can.